### DISCUSSION TITLE
240311

### DISCUSSION THREAD

**Discussion by caseV12** [codeit-part4-8team-project/main#31] [팀회의]
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- 자잘한 컨벤션에 관한 부분 짚고 넘어가면 좋을 것 같습니다.  (eg. 디렉토리, 폴더 구조, 이미지 변수명 등)

## 어제 한 일 (주말)
- 칸반보드, 사이드바, 공지사항 UI 구현

## 오늘 할 일
- 사이드바 컨텍스트로 관리
- 전체 페이지 레이아웃

## 오늘의 일정 (코어 타임)
- 없음

## 멘토님 질문 사항
-

**Comment by caseV12** [comment]
## 논의사항
필겸님이 토요일에 말씀하신 모달 논의
## 어제 한 일
유저 기능 쪽에서 리프레시토큰 리다이렉트 구현
## 오늘 할 일
로그인 페이지 전체 완성
## 오늘의 일정 (코어 타임)
없음.
## 멘토님 질문 사항

**Comment by sobokki** [comment]
## 논의사항
필겸님 채빈님 모달 논의사항 저도 함께 봐야할 것 같습니다!
## 어제 한 일
네브바 UI변경
## 오늘 할 일
캘린더 새로운 디자인 변경, 모달기능 및 세부 기능 추가 
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
pr 올린거 궁금한게 몇개 있어서 여쭤볼게 있을것 같아요!
## 어제 한 일
팀멤버리스트,멤버조회모달창 
## 오늘 할 일
....
## 오늘의 일정 (코어 타임)
없습니다
## 멘토님 질문 사항
차차 논의사항들 얘기 해보며 만들어보면 괜찮을것 같아요

**Comment by jisunipark** [comment]
- 2시에 채빈님 모달 훅 설명
- 디렉토리, 폴더 구조, 이미지 변수명
  - 컴포넌트: 파스칼 케이스 ex. BoardList
  - 디렉토리: 카멜케이스 ex. boardList
  - 이미지 변수명: 파스칼 케이스 ex. ClockIcon
  - 테일윈드 스타일 변수: 카멜 케이스
